{ name: 'MongoTimeoutError',
  reason:
   { MongoError: Authentication failed.
       at Function._getError (/backend/node_modules/mongodb/lib/core/auth/scram.js:125:14)
       at sendAuthCommand (/backend/node_modules/mongodb/lib/core/auth/scram.js:175:31)
       at Connection.messageHandler (/backend/node_modules/mongodb/lib/core/connection/connect.js:334:5)
       at Connection.emit (events.js:198:13)
       at processMessage (/backend/node_modules/mongodb/lib/core/connection/connection.js:368:10)
       at Socket.<anonymous> (/backend/node_modules/mongodb/lib/core/connection/connection.js:537:15)
       at Socket.emit (events.js:198:13)
       at addChunk (_stream_readable.js:288:12)
       at readableAddChunk (_stream_readable.js:269:11)
       at Socket.Readable.push (_stream_readable.js:224:10)
       at TCP.onStreamRead [as onread] (internal/stream_base_commons.js:94:17)
     name: 'MongoNetworkError',
     errorLabels: [ 'TransientTransactionError' ],
     [Symbol(mongoErrorContextSymbol)]: {} },
  level: 'error',
  message:
   'MongoDB connection error: Server selection timed out after 30000 ms',
  stack:
   'MongoTimeoutError: Server selection timed out after 30000 ms\n    at Timeout.setTimeout (/backend/node_modules/mongodb/lib/core/sdam/topology.js:878:9)\n    at ontimeout (timers.js:436:11)\n    at tryOnTimeout (timers.js:300:5)\n    at listOnTimeout (timers.js:263:5)\n    at Timer.processTimers (timers.js:223:10)',
  timestamp: '2019-11-12T19:02:47.455Z',
  [Symbol(mongoErrorContextSymbol)]: {} }
